### PR TITLE
More functionality on home page

### PR DESCRIPTION
**Pull request by Skellyboy38**
(read the commit messages lol)

### CODE REVIEW

File: app/static/js/app.js [hunk 1]
```diff
@@ -273,11 +259,11 @@ $( document ).ready(function() {
 });
 
 function hoverInLogo(hoveredOver){
-	document.getElementById("active").id="oldActive";
+	//document.getElementById("active").id="oldActive";
```
Comment (Progaine) [inline app/static/js/app.js]: This line is required for the feature to work.

File: app/static/js/profile.js [hunk 2]
```diff
@@ -45,3 +45,37 @@ $(function () {
         }
     });
 });
+
+$( document ).ready(function()
+{
+    $.ajax({ //get all the student's fall
```
Comment (Progaine) [inline app/static/js/profile.js]: I think this should be made into it's own function.

File: app/static/js/app.js (hunk 1 above)
Comment (Skellyboy38) [inline app/static/js/app.js]: Ah ok, I removed it because it was throwing null errors. I'll put it back but maybe u can take a look at it.

Comment (ddayguerrero) [inline app/static/js/app.js]: Is this fixed and ready to merge?

Comment (Skellyboy38) [inline app/static/js/app.js]: Yeah you can merge it darrel. The line I commented out does not cause the system to crash or anything, it just throws an annoying message in the console

**Pull request merged**